src/defaultTestResultHandler.js

```diff
@@ -102,48 +156,84 @@ function defaultTestResultHandler(config, testResult) {
   }
   */
 
-  console.log(_getResultHeader(allTestsPassed, pathStr, [
+  var allTestsPassed = testResult.numFailingTests === 0;
+
+  console.log(_getResultHeader(
+    allTestsPassed, filePath, [
     testRunTimeString
   ]));
```

Comment:
This indentation is weird. It wasn't awesome to begin with but it's worse now. Do this instead:

```js
console.log(_getResultHeader(
  allTestsPassed,
  filePath,
  [testRunTimeString]
));
```